hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/DMROperationAttributes.java

```diff
@@ -40,8 +40,9 @@
                     .addFlag(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
                     .build();
 
+
     AttributeDefinition[] ATTRIBUTES = {
             PATH,
-            OPERATION_NAME
+            OPERATION_NAME,
```

Comment:
remove comma - btw: this OPERATION_NAME is the thing we should rename to something like INTERNAL_NAME